pxtwinrt/hiddeploy.ts

```diff
@@ -68,6 +68,7 @@ namespace pxt.winrt {
                 });
             }, Promise.resolve<Windows.Devices.Enumeration.DeviceInformationCollection>(null));
 
+            let deviceId: string;
```

Comment:
Do you clear this value?

Reply:
It's a local variable to the function, only used to get the device status in case of an error, not used outside of this function